### ISSUE TITLE
Items that link to GitHub do not load in reader view

### ISSUE THREAD

**Issue by blackketter** [Ranchero-Software/NetNewsWire#1690]
Items that link directly to GitHub do not load properly in Reader view with the message:  

> You can’t perform that action at this time.
> 
>  You signed in with another tab or window. Reload to refresh your session. You signed out in another tab or window. Reload to refresh your session.

Found today in the Hacker News RSS feed http://news.ycombinator.com/rss
with the link to https://github.com/neeilan/neeilang

**Comment by brentsimmons** [comment]
I don’t think we can do anything about that. Maybe disable Reader view when it’s a GitHub link?

**Comment by blackketter** [comment]
Not sure what's going on here.  That link does load correctly in a Safari private tab.  From the message it sounds like there's some cookie or other credential that is confusing GitHub.  This concerns me as it seems that there's some information leakage between reader sessions.

**Comment by brentsimmons** [comment]
The Reader View is a service run by the generous folks at Feedbin and provided to NetNewsWire users for free. We don’t write or maintain that code. I recommend reporting bugs with reader view to the Feedbin folks. Thanks!
